### DISCUSSION TITLE
Best practices for preexisting conditions and medical history

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1443] [MIMIC-General] [answered]
Hello Mimic-Community, 

me and my team are working on some questions regarding organ dysfunction on the ICU. One parameter of high interest for us are preexisting conditions that are already manifest in a patient on ICU admission, e.g. diabetes or chronic kidney disease. I was thinking about using coded diagnoses in Mimic-ED for that purpose or checking for medications on admission. However it says that this table contains only the ten most relevant diagnoses during the hospital stay and I fear not all preexisting conditions fall under that premise. 

I was wondering if there are any different approaches that I did not yet think about? Are there any best practices in place?

Kind regards

**Comment by alistairewj** [comment, marked as answer]
With the release of the [MIMIC-IV Clinical Notes](https://physionet.org/content/mimic-iv-note/), your best bet is the Past Medical History section of the discharge summaries. I do have some code kicking around somewhere that sections the data, and I'll try to put that into the repo sometime soon (but feel free to contribute your own, if you have it).

The MIMIC-IV-ED diagnoses are not great for this purpose, as the reason for an admission to the ED may be the same as it is for the ICU. There's no guarantee that the ED diagnoses are chronic and/or pre-existing to the hospital stay.
